pkg/artifactory/resource_artifactory_webhook_release_bundle.go

```diff
@@ -14,13 +15,13 @@ type ReleaseBundleWebhookCriteria struct {
 }
 
 var releaseBundleWebhookSchema = func(webhookType string) map[string]*schema.Schema {
-	return mergeSchema(baseWebhookBaseSchema(webhookType), map[string]*schema.Schema{
+	return utils.MergeSchema(baseWebhookBaseSchema(webhookType), map[string]*schema.Schema{
```

Comment:
`1.18` has `Copy` - does it have an array like ours? Should be it it or ours?

Reply:
1.18's `Copy` only take one map as source. `MergeSchema` takes variadic so we can't replace wholesale.